core/templates/dev/head/domain/exploration/AnswerStatsObjectFactory.ts

```diff
@@ -17,7 +17,7 @@
  * particular answer from some particular state.
  */
 
-import * as cloneDeep from 'lodash/cloneDeep';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Please make sure to test this. This can become tricky at times.

Reply:
Sure, will do. What issues did you face here (It only interfered with Karma tests or with the dev server as well?)

Reply:
@YashJipkate could you please reply to this?

Reply:
So sorry, I somehow missed this.

> What issues did you face here (It only interfered with Karma tests or with the dev server as well?)

Yes, those were only karma-specific, worked fine on dev server.

Reply:
Ok, I checked by creating and publishing a simple exploration on dev server and it worked fine. Karma tests passed too!